lib/ButtonNew/ButtonPrimary.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import cx from 'classnames';
+import { ButtonBase } from './ButtonBase';
+import { propTypes, defaultProps, sizes } from './common';
+
+function ButtonPrimary({
+  children,
+  className,
+  disabled,
+  connectedRight,
+  danger,
+  ...rest
+}) {
+  const classes = cx('text-white ', className, {
+    'bg-blue-primary': !danger,
+    'bg-red-primary': danger,
```

Comment:
💚Would it be better to have a separate components for BluePrimary and RedPrimary?

Reply:
I like this idea. `ButtonPrimary` and `ButtonPrimaryDanger` certainly make the components easier to read 👍